src/Symfony/Component/Notifier/Bridge/LineBot/LineBotTransportFactory.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\LineBot;
+
+use Symfony\Component\Notifier\Exception\IncompleteDsnException;
+use Symfony\Component\Notifier\Exception\UnsupportedSchemeException;
+use Symfony\Component\Notifier\Transport\AbstractTransportFactory;
+use Symfony\Component\Notifier\Transport\Dsn;
+
+/**
+ * @author Yi-Jyun Pan <me@pan93.com>
+ */
+final class LineBotTransportFactory extends AbstractTransportFactory
+{
+    private const SCHEME = 'linebot';
+
+    protected function getSupportedSchemes(): array
+    {
+        return [self::SCHEME];
+    }
+
+    public function create(Dsn $dsn): LineBotTransport
+    {
+        if (self::SCHEME !== $dsn->getScheme()) {
+            throw new UnsupportedSchemeException($dsn, self::SCHEME, $this->getSupportedSchemes());
+        }
+
+        $accessToken = $dsn->getUser();
+        if (null === $accessToken) {
+            throw new IncompleteDsnException('Missing channel access token.', 'linebot://'.$dsn->getHost());
+        }
```

Comment:
```suggestion
        $accessToken = $this->getUser();
```

Reply:
I still prefer this check. `$dsn->getUser()` returns `null` if the DSN does not contain a username. Since an access token is required for this bridge, it may throw a different type of violation message that is difficult to debug. Some users even interpret this as an error.

![image](https://github.com/user-attachments/assets/20756558-80cf-425f-a602-f299c425e96b)

However, the existing notifiers do not check for this. I'm unsure how we can balance maintaining the current coding style while providing a helpful hint for this situation.

Reply:
Not dsn->getUser, but THIS->getUser

It does the check internally

Reply:
Awesome. Great to know!

Reply:
Fixed.